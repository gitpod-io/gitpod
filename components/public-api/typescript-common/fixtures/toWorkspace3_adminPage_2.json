{
  "organizationId": "organizationId",
  "ownerId": "ownerId",
  "contextURL": "https://github.com/gitpod-io/empty",
  "projectId": null,
  "description": "empty",
  "context": {
    "isFile": false,
    "path": "",
    "title": "gitpod-io/empty - master",
    "ref": "master",
    "refType": "branch",
    "revision": "revision",
    "repository": {
      "cloneUrl": "https://github.com/gitpod-io/empty.git",
      "host": "github.com",
      "name": "empty",
      "owner": "gitpod.io",
      "private": false
    }
  },
  "config": {
    "image": "gitpod/workspace-base",
    "ports": [
      {
        "port": 8080,
        "onOpen": "notify"
      },
      {
        "port": 8081,
        "onOpen": "ignore"
      },
      {
        "port": 8082,
        "onOpen": "ignore"
      },
      {
        "port": 8083,
        "onOpen": "ignore"
      },
      {
        "port": 3306,
        "onOpen": "ignore"
      }
    ],
    "tasks": [
      {
        "init": "npm install",
        "command": "/etc/docker/startup.sh"
      }
    ],
    "vscode": {
      "extensions": [
      ]
    }
  },
  "imageSource": {
    "baseImageResolved": "gitpod/workspace-base"
  },
  "imageNameResolved": "gitpod/workspace-base",
  "baseImageNameResolved": "gitpod/workspace-base",
  "shareable": false,
  "type": "regular",
  "softDeleted": null,
  "deleted": false,
  "pinned": true,
  "basedOnPrebuildId": null,
  "basedOnSnapshotId": null,
  "workspaceId": "longlong-650a-413a-b5ed-aaaaa",
  "region": "us02",
  "startedTime": "2019-12-12T18:38:38.625Z",
  "deployedTime": "2019-12-12T18:38:21.501Z",
  "stoppedTime": "2019-12-12T19:20:11.821Z",
  "ideUrl": "https://longlong-650a-413a-b5ed-aaaaa.ws-us02.gitpod.io",
  "workspaceImage": "gitpod/workspace-base",
  "status": {
    "repo": {
      "branch": "master",
      "latestCommit": "latestCommit",
      "untrackedFiles": [
        "a.jpg"
      ],
      "uncommitedFiles": [
        "package.json"
      ]
    },
    "phase": "stopped",
    "conditions": {
      "timeout": "workspace timed out after period of inactivity took longer than 00h30m",
      "deployed": false,
      "pullingImages": false,
      "serviceExists": false
    },
    "exposedPorts": [
      {
        "port": 8080,
        "targetPort": 38080
      },
      {
        "port": 8081,
        "targetPort": 38081
      },
      {
        "port": 8082,
        "targetPort": 38082
      },
      {
        "port": 8083,
        "targetPort": 38083
      },
      {
        "port": 3306,
        "targetPort": 33306
      }
    ]
  },
  "gitStatus": null,
  "phasePersisted": "stopped",
  "configuration": null,
  "imageBuildInfo": null,
  "instanceId": "instanceId",
  "workspaceCreationTime": "2019-08-08T20:38:43.425Z",
  "instanceCreationTime": "2019-12-12T18:38:21.095Z",
  "phase": "stopped"
}
