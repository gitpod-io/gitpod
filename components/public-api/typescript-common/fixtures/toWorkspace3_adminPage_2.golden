{
    "result": {
        "id": "longlong-650a-413a-b5ed-aaaaa",
        "metadata": {
            "ownerId": "ownerId",
            "organizationId": "organizationId",
            "configurationId": "",
            "annotations": {},
            "name": "empty",
            "pinned": true,
            "originalContextUrl": "https://github.com/gitpod-io/empty"
        },
        "spec": {
            "initializer": {
                "specs": [
                    {
                        "git": {
                            "remoteUri": "https://github.com/gitpod-io/empty.git",
                            "upstreamRemoteUri": "",
                            "targetMode": "CLONE_TARGET_MODE_REMOTE_COMMIT",
                            "cloneTarget": "revision",
                            "checkoutLocation": "",
                            "config": {
                                "customConfig": {},
                                "authentication": "AUTH_METHOD_UNSPECIFIED",
                                "authUser": "",
                                "authPassword": "",
                                "authOts": ""
                            }
                        }
                    }
                ]
            },
            "type": "WORKSPACE_TYPE_REGULAR",
            "ports": [
                {
                    "port": "8080",
                    "admission": "ADMISSION_LEVEL_OWNER_ONLY",
                    "url": "",
                    "protocol": "PROTOCOL_HTTP"
                },
                {
                    "port": "8081",
                    "admission": "ADMISSION_LEVEL_OWNER_ONLY",
                    "url": "",
                    "protocol": "PROTOCOL_HTTP"
                },
                {
                    "port": "8082",
                    "admission": "ADMISSION_LEVEL_OWNER_ONLY",
                    "url": "",
                    "protocol": "PROTOCOL_HTTP"
                },
                {
                    "port": "8083",
                    "admission": "ADMISSION_LEVEL_OWNER_ONLY",
                    "url": "",
                    "protocol": "PROTOCOL_HTTP"
                },
                {
                    "port": "3306",
                    "admission": "ADMISSION_LEVEL_OWNER_ONLY",
                    "url": "",
                    "protocol": "PROTOCOL_HTTP"
                }
            ],
            "environmentVariables": [],
            "git": {
                "username": "",
                "email": ""
            },
            "admission": "ADMISSION_LEVEL_OWNER_ONLY",
            "class": "",
            "sshPublicKeys": [],
            "subassemblyReferences": [],
            "logUrl": ""
        },
        "status": {
            "statusVersion": "1576178411",
            "phase": {
                "name": "PHASE_STOPPED",
                "lastTransitionTime": "2019-12-12T19:20:11.821Z"
            },
            "workspaceUrl": "https://longlong-650a-413a-b5ed-aaaaa.ws-us02.gitpod.io",
            "conditions": {
                "failed": "",
                "failedReason": "FAILED_REASON_UNSPECIFIED",
                "timeout": "workspace timed out after period of inactivity took longer than 00h30m"
            },
            "gitStatus": {
                "cloneUrl": "https://github.com/gitpod-io/empty.git",
                "branch": "master",
                "latestCommit": "revision",
                "uncommitedFiles": [],
                "totalUncommitedFiles": 0,
                "untrackedFiles": [],
                "totalUntrackedFiles": 0,
                "unpushedCommits": [],
                "totalUnpushedCommits": 0
            },
            "instanceId": "instanceId"
        }
    },
    "err": ""
}
